File: old/NumberGameTest1/Player.cs
using System.Collections.Generic;

namespace NumberGameTest1
{
    class Player
    {
        private string name;
        public string Name
        {
            get
            {
                return name;
            }
            set
            {
                if (!string.IsNullOrWhiteSpace(value))
                {
                    name = value;
                }
            }
        }
        public List<Tile> Tiles { get; set; } = new List<Tile>();
        public bool IsPlaying { get; set; } = true;

        public Player(string name)
        {
            Name = name;
        }
    }
}

File: src/uk/co/ruben9922/numbergame/Player.java
package uk.co.ruben9922.numbergame;

import java.util.LinkedList;
import java.util.List;

class Player {
    private String name;
    private List<Tile> tiles;

    public Player(String name, List<Tile> tiles) {
        this.name = name;
        this.tiles = tiles;
    }

    public Player(String name) {
        this(name, new LinkedList<>());
    }

    public String getName() {
        return name;
    }

    public List<Tile> getTiles() {
        return tiles;
    }

    public void setTiles(List<Tile> tiles) {
        this.tiles = tiles;
    }

    public void printTiles() {
        System.out.format("%s's tiles:\n", name);
        ListUtilities.printList(tiles, true);
    }
}
